web/models.py

```
@@ -193,6 +193,54 @@ def save(self, *args, **kwargs):
+    def __str__(self):
+        return self.title
+
+
+class Question(models.Model):
+    QUESTION_TYPES = [
+        ("mcq", "Multiple Choice"),
+        ("checkbox", "Checkbox (Multiple Answers)"),
+        ("text", "Text Answer"),
+        ("true_false", "True/False"),
+        ("scale", "Scale Rating"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)